runtime/src/dispatcher.rs

```diff
@@ -303,7 +334,8 @@ impl Dispatcher {
         let (mut outputs, mut tags, messages) =
             StorageContext::enter(&mut cache.mkvs, untrusted_local.clone(), || {
                 txn_dispatcher.dispatch_batch(&inputs, txn_ctx)
-            });
+            })
+            .unwrap();
```

Comment:
Do I understand this correctly that the runtime will panic when a batch is aborted? This doesn't seem right? It should just propagate the error back, saying that the batch was aborted.

Reply:
fixed